### PR TITLE
Change logged out header links

### PR DESCRIPTION
**Pull request by thethomaseffect**
Overwrites the hover styles from parent component until that's refactored to not use inline styles. Change to clicking "disabled" links to open a modal popup on hold depending on some refactoring of required components from other repos, for now the link has simply been changed to go to signup.

### CODE REVIEW

File: src/Component/Molecule/SiteNavBarLoggedOut/SiteNavBarLoggedOut.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    border-left: .1rem solid #b3b3b3;
+  }
+`;
+
+const SiteNavBarLoggedOut = ({
+  showVerticalNav,
+}: SiteNavBarLoggedOutProps) => (
+  <HeaderCenter>
+    <HeaderNav showVerticalNav={showVerticalNav}>
+      <a id="header-nav-solutions" href="/solutions/">
```
Comment (huwd) [inline src/Component/Molecule/SiteNavBarLoggedOut/SiteNavBarLoggedOut.jsx]: Do these even need an Href if they'll never actually refer?

File: src/HighOrderComponent/SimpleList/SimpleListTest.jsx [hunk 2]
```diff
@@ -24,7 +24,9 @@ describe('SimpleList High Order Component', () => {
         </SimpleList>,
       );
 
-      expect(simpleListComponent.find('ul').find('li').first().text()).to.contain('testItem');
+      expect(
+        simpleListComponent.find('ul').find('li').first().text(),
```
Comment (huwd) [inline src/HighOrderComponent/SimpleList/SimpleListTest.jsx]: Cool, been wondering how to do that - nice!

**Comment by huwd** [comment]
Humm, this doesn't seem to be working for me - i've got this branch of styleguide linked with lists, getting this:

peek 2018-09-25 12-35

File: src/Component/Molecule/SiteNavBarLoggedOut/SiteNavBarLoggedOut.jsx (hunk 1 above)
Comment (Floppy) [reply src/Component/Molecule/SiteNavBarLoggedOut/SiteNavBarLoggedOut.jsx]: yeah, best to be semantically correct

Comment (thethomaseffect) [reply src/Component/Molecule/SiteNavBarLoggedOut/SiteNavBarLoggedOut.jsx]: They could be changed but it wouldn't be any "better" so no point

**Pull request merged**